Services/UI/OJS.Services.Ui.Business/Implementations/ContestsBusinessService.cs

```diff
@@ -190,6 +193,12 @@ public async Task ValidateContestPassword(int id, bool official, string password
                     .FirstOrDefault();
             });
 
+            var practiceContestParticipantsViewModel = await this.contestParticipantsCacheService.GetPracticeContestParticipantsCount(model.ContestId);
+            participationModel.PracticeParticipantsCount = practiceContestParticipantsViewModel.PracticeParticipantsCount;
+
+            var competeContestParticipantsViewModel = await this.contestParticipantsCacheService.GetCompeteContestParticipantsCount(model.ContestId);
+            participationModel.CompeteParticipantsCount = competeContestParticipantsViewModel.CompeteParticipantsCount;
```

Comment:
Why get both compete and practice, when the user enters in the contest? We should display only one of the values.

Reply:
Reworked.